### PR TITLE
new token build pipeline

### PR DESCRIPTION
**Pull request by fschade**
- introduce token transformations for colors (lighten, brighten, ...)
- cleanup doc components and show css variables, other color formats, ... in templates
- clean up tokens
- clean up sass variables
- remove unused styles
- auto generate css variables, no manual mapping needed anymore
- split up tokens for docs and ods
- use css variables where ever possible
- restructure webpack build pipeline
- fix failing tests
- introduce new style token build pipeline

### CODE REVIEW

**Comment by pascalwengerter** [comment]
Took care of rebasing to master 👌🏽

File: src/tokens/ods/breakpoint.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "value": "1599px",
+        "attributes": {}
+      }
+    },
+    "xlarge": {
+      "old": "breakpoint-xlarge",
+      "value": "1600px",
+      "attributes": {}
+    },
+    "xsmall-": {
```
Comment (pascalwengerter) [inline src/tokens/ods/breakpoint.json:44]: Is the dash in the name intended?

File: src/styles/theme/oc-visibility.scss [hunk 2]
```diff
@@ -9,28 +9,28 @@
...
-@media (min-width: $breakpoint-large) {
+@media (min-width: $oc-breakpoint-large-default) {
   .oc-hidden\@l {
     display: none !important;
   }
 }
 
 /* Large screen and bigger */
-@media (min-width: $breakpoint-xlarge) {
+@media (min-width: $oc-breakpoint-xlarge) {
```
Comment (pascalwengerter) [inline src/styles/theme/oc-visibility.scss:33]: Why no `-default` in this case?

File: src/components/OcTextarea.vue [hunk 3]
```diff
@@ -212,7 +212,7 @@ export default {
     @extend .uk-flex-middle;
     @extend .oc-mt-xs;
 
-    min-height: $global-font-size * 1.5;
+    min-height: $oc-font-size-default * 1.5;
```
Comment (pascalwengerter) [inline src/components/OcTextarea.vue:215]: Why are we using a SASS variable here?

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Looks pretty neat already, got few questions in the comments, the linter complains about some ordering in the `/docs/` styles and waiting for the naming adjustments we discussed to be added

File: src/styles/theme/oc-visibility.scss (hunk 2 above)
Comment (fschade) [reply src/styles/theme/oc-visibility.scss:33]: only variables with variations have a default flag.

File: src/tokens/ods/breakpoint.json (hunk 1 above)
Comment (fschade) [reply src/tokens/ods/breakpoint.json:44]: thanks :)

File: src/components/OcTextarea.vue (hunk 3 above)
Comment (fschade) [reply src/components/OcTextarea.vue:215]: i had problems to mix css variables and sass calculations

**Review by pascalwengerter — APPROVED** [review]
LGTM ✊🏽

**Review by kulmann — CHANGES_REQUESTED** [review]
awesome! 🚀  only found super tiny nitpicks. ;-)

**Review by kulmann — APPROVED** [review]
LGTM 🚀

**Pull request merged**